Tools/ardupilotwaf/boards.py

```diff
@@ -126,6 +129,8 @@ def configure_env(self, cfg, env):
             '-Werror=uninitialized',
             '-Werror=init-self',
             '-Wfatal-errors',
+            '-DUAVCAN_CPP_VERSION=UAVCAN_CPP03',
+            '-DUAVCAN_NO_ASSERTIONS',
```

Comment:
This is a global for all boards. It is harmless for boards that do not have UAVCAN, but is common for all and will remove the need to do this for every board.